MonoGame.Framework/Platform/Graphics/Texture3D.OpenGL.cs

```diff
@@ -14,9 +14,9 @@ public partial class Texture3D : Texture
 
         private void PlatformConstruct(GraphicsDevice graphicsDevice, int width, int height, int depth, bool mipMap, SurfaceFormat format, bool renderTarget)
         {
-#if GLES
-            throw new NotSupportedException("OpenGL ES 2.0 doesn't support 3D textures.");
-#else
+            if (GL.BoundApi == GL.RenderApi.ES && graphicsDevice.glMajorVersion < 3)
```

Comment:
Same here... maybe add `GraphicsCapabilities.SupportsTexture3D` to simplify this code.